axis_backend/apps/clients/services/__init__.py

```
@@ -0,0 +1,5 @@
+"""Service layer for Clients app."""
+from .industry_service import IndustryService
+from .client_service import ClientService
+
+__all__ = ['IndustryService', 'ClientService']
```

Ruff (0.14.5)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)